src/index.html

```diff
@@ -34,158 +35,219 @@
-        style="
-          border-bottom-color: var(--wp--preset--color--tertiary);
-          border-bottom-style: solid;
-          border-bottom-width: 1px;
-          padding-bottom: var(--wp--preset--spacing--50);
-        "
-      >
-        <h3 class="wp-block-heading has-medium-font-size">
-          <strong>How much does it cost?</strong>
-        </h3>
-
-        <p>
-          We charge $400 for access to ~100 hours with our virtual AI
-          instructor. But we&#8217;re currently offering a 25% early-bird
-          discount &#8211; so $300 for a limited time.
-        </p>
-
-        <p>
-          We&#8217;ll give you a 100% refund if you&#8217;re not satisfied in
-          your first 14 days, no questions asked.
-        </p>
-      </div>
-    </div>
+    <h3 class="wp-block-heading has-small-font-size">Do I get a certificate of graduation from this?</h3>
+    <p>No. Our software is geared to help you ace your checkride. You'll need to work with your CFI or online ground school to get the certificate that allows you to take the private pilot written test.</p>
 
-    <div class="wp-block-column is-layout-flow wp-block-column-is-layout-flow">
-      <div
-        class="wp-block-group is-vertical is-layout-flex wp-container-20 wp-block-group-is-layout-flex"
-        style="
-          border-bottom-color: var(--wp--preset--color--tertiary);
-          border-bottom-style: solid;
-          border-bottom-width: 1px;
-          padding-bottom: var(--wp--preset--spacing--50);
-        "
-      >
-        <h3 class="wp-block-heading has-medium-font-size">
-          <strong>Does this replace a real instructor?</strong>
-        </h3>
-
-        <p>
-          No! You&#8217;ll still need your Certified Flight Instructor to
-          prepare you for the practical portion of the exam. We help you prepare
-          for the oral section so you can spend less time on the ground and more
-          time in the air.
-        </p>
-      </div>
+    <h3 class="wp-block-heading has-small-font-size">Do I need to download an app?</h3>
+    <p>No. Checkride AI runs in your browser, so you don't have to download or install anything.</p>
 
-      <h3 class="wp-block-heading has-medium-font-size">
-        Do I get a certificate of graduation from this?
-      </h3>
+    <h3 class="wp-block-heading has-small-font-size">Which pilot licenses does this work for?</h3>
+    <p>Private pilot license. Instrument and commercial checkride prep coming soon.</p>
 
+  </div>
+</div>
+<style>
+  /* Responsive adjustments for smaller screens */
+  @media screen and (max-width: 768px) {
+    .wp-block-group {
+      flex-direction: column;
+    }
+    .wp-block-column {
+      /* Adjust flex-basis for full width on small screens */
+      flex-basis: 100%;
+      padding-left: 20px; /* Adjust padding as necessary */
+      padding-right: 20px; /* Adjust padding as necessary */
```

Comment:
```suggestion
    .wp-block-column {
      /* Adjust flex-basis for full width on small screens */
      flex-basis: 100% !important;
      padding-left: 20px !important; /* Adjust padding as necessary */
      padding-right: 20px !important; /* Adjust padding as necessary */
```
Because of how css is processed, you'll need to add the `!important` to get this to show up in mobile